task/sast-coverity-check/0.3/migrations/0.3.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# Created for task: sast-coverity-check@0.3
+# Creation time: $(date -u +"%Y-%m-%dT%H:%M:%SZ")
+
+declare -r pipeline_file=${1:?missing pipeline file}
+
+# Determine the correct image-digest and image-url values based on task presence
+if yq -e '.spec.tasks[] | select(.name == "build-oci-artifact")' "$pipeline_file" >/dev/null; then
+    image_digest_value="$(yq '.tasks[] | select(.name == "build-oci-artifact").results[] | select(.name == "IMAGE_DIGEST").value' "$pipeline_file")"
+    image_url_value="$(yq '.tasks[] | select(.name == "build-oci-artifact").results[] | select(.name == "IMAGE_URL").value' "$pipeline_file")"
+elif yq -e '.spec.tasks[] | select(.name == "build-image-index")' "$pipeline_file" >/dev/null; then
+    image_digest_value="$(yq '.tasks[] | select(.name == "build-image-index").results[] | select(.name == "IMAGE_DIGEST").value' "$pipeline_file")"
+    image_url_value="$(yq '.tasks[] | select(.name == "build-image-index").results[] | select(.name == "IMAGE_URL").value' "$pipeline_file")"
```

Comment:
In addition, there are missing `.spec` in above 4 YAML path.

Reply:
yes, 
missed by mistake, fixed in all places